pkg/lightning/mydump/aurora_test.go

```
@@ -0,0 +1,123 @@
+		{
+			name:     "escaped identifiers",
+			path:     "my%20db/my%20db.order%2Eitems/1/part-a.parquet",
+			matched:  true,
+			database: "my db",
+			schema:   "my db",
+			table:    "order.items",
+			form:     AuroraSnapshotPathFormBatched,
+		},
+		{
+			name:       "dotted export prefix",
+			path:       "prefix/customer.v1/export-123/db/db.users/1/part-00000-a.parquet",
```

golangci-lint (2.12.2)

[warning] line-length-limit: line is 91 characters, out of limit 80

(revive)